README.md

```diff
@@ -1,7 +1,7 @@
 # JSONs to Looker views (J2V)
 
 J2V is a simple command-line tool to convert JSON to [Looker](https://looker.com/) readable files in forms of [Looker Views](https://docs.looker.com/reference/view-params/view) and [Looker Explores](https://docs.looker.com/reference/explore-params/explore).
-
+Also it outputs an SQL with proper paths and explosion expressions.
```

Comment:
Add an empty line before and after this sentence, which will make it a new paragraph on the rendered output. Otherwise, the 3 intended paragraphs will merge into one.

Reply:
```suggestion

Also it outputs an SQL with proper paths and explosion expressions.

```